projects/netgrif-components/src/lib/tabs/tab-view/tab-view.component.scss

```diff
@@ -17,7 +17,7 @@
 //.tab-group mat-tab-group {
 //    flex-grow: 1;
 //
-//    ///deep/ .mat-tab-body-wrapper {
```

Comment:
why not just get rid of this css?

Reply:
removed, if it will be needed it will be in git history